src/Routes/ImageManagerDetail/DetailsHeader.js

```diff
@@ -10,45 +11,35 @@ import {
   BreadcrumbItem,
 } from '@patternfly/react-core';
 import StatusLabel from './StatusLabel';
-import { useSelector, shallowEqual } from 'react-redux';
 import { routes as paths } from '../../../package.json';
 
-const DetailsHead = () => {
-  const { isLoading, hasError, data } = useSelector(
-    ({ imageStatusReducer }) => ({
-      isLoading:
-        imageStatusReducer?.isLoading !== undefined
-          ? imageStatusReducer?.isLoading
-          : true,
-      hasError: imageStatusReducer?.hasError || false,
-      data: imageStatusReducer?.data || null,
-    }),
-    shallowEqual
-  );
-
-  const status = !isLoading && !hasError ? data.Status : null;
-
+const DetailsHead = ({ name, status }) => {
   return (
     <>
       <Breadcrumb>
         <BreadcrumbItem>
           <Link to={paths['manage-images']}>Manage Images</Link>
         </BreadcrumbItem>
-        <BreadcrumbItem isActive>{data?.Name}</BreadcrumbItem>
+        <BreadcrumbItem isActive>{name}</BreadcrumbItem>
       </Breadcrumb>
 
       <TextContent>
         <TextList component="dl">
           <TextListItem component="h1" className="grid-align-center">
-            {data?.Name}
+            {name}
           </TextListItem>
           <TextListItem className="pf-u-pt-xs" component="dd">
-            {isLoading ? <Skeleton /> : <StatusLabel status={status} />}
+            {status ? <StatusLabel status={status} /> : <Skeleton />}
```

Comment:
Not a fan of this. Can we be more explicit about this. Maybe like `status === 'loading' ? <Skeleton /> : ... `